app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,131 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.enums.HttpMethodType;
+import com.codesoom.assignment.error.ClientError;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.service.TaskService;
+import com.codesoom.assignment.utils.JsonUtil;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class AppHttpHandler implements HttpHandler {
+    private final TaskService taskService = new TaskService();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        HttpMethodType method = HttpMethodType.valueOf(exchange.getRequestMethod());
+        String[] pathArr = exchange.getRequestURI().getPath().split("/");
+        String path = pathArr[1];
+        Long userId = getUserId(pathArr);
+        String requestBody = new BufferedReader(new InputStreamReader(exchange.getRequestBody()))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        if (!path.equals("tasks")) {
+            ClientError.notFound(exchange);
```

Comment:
이 `notFound` 메소드 안쪽에서 예외를 던지는 방식으로 흐름 제어를 하고 있을 것 같군요. 이렇게 하면 `notFound`의 구현을 모르는 사람이 `handle` 메소드를 읽을 때 무척 혼란스럽게 됩니다. 그렇게 예외를 핵심으로 제어를 하지 않는 것이 좋습니다. 만약 그 방법을 그대로 사용하고 싶다면 차라리 이렇게 return을 눈에 보이게 작성해주는 쪽이 handle 메소드를 읽는 사람에게 도움이 될 거에요.

```suggestion
            ClientError.notFound(exchange);
            return;
```

Reply:
(350ab66) 예외를 던지는 방식에서 직관적으로 보이도록 return으로 수정하였습니다!
감사합니다 :)